.eslintrc.js

```diff
@@ -1,13 +1,23 @@
 module.exports = {
   root: true,
+  env: {
+    browser: true,
+    node: true
+  },
   parserOptions: {
     parser: 'babel-eslint',
     sourceType: 'module'
   },
   extends: [
-    '@nuxtjs'
+    '@nuxtjs/eslint-config-typescript',
+    'prettier',
+    'prettier/vue',
+    'plugin:prettier/recommended',
+    'plugin:nuxt/recommended'
   ],
+  plugins: ['prettier'],
+  // add your custom rules here
   rules: {
-    "space-before-function-paren": ["error", "never"],
+    // 'space-before-function-paren': ['error', 'never'],
```

Comment:
This can probably be deleted altogether. the spacing conventions are probably set in one of the recommended linters above.

Reply:
the idea would be that having at least 1 rule would make thing easier for users that would have different preferences when it comes to editor / linting rules

Reply:
ok. I think we might also want the comma-dangle rule in here: 
```
"comma-dangle": ["error", "never"],
```